config/ide-helper.php

```diff
@@ -292,6 +292,17 @@
     */
     'force_fqn' => false,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Use generics syntax
+    |--------------------------------------------------------------------------
+    |
+    | Use generics syntax within DocBlocks,
+    | e.g. `Collection<User>` instead of `Collection|User[]`.
+    |
+    */
+    'use_generics_annotations' => false,
```

Comment:
I have not yet looked into detail of this PR and will do it later, but if it works really good I'm also tempted to suggesting to make it even default to `true`. But that was just a quick showerthought :)

Reply:
That would be my preference too, but I thought this was a decision best left to you. I originally had the fallback value as `version_compare($this->laravel['version'], '9', '>=')` (i.e. `true` if Laravel version is at least version 9), but was in two minds about it. In the end, [I removed it](https://github.com/barryvdh/laravel-ide-helper/pull/1298/commits/04cc0ae9762878dac8cf8e2580714412d3da3e76). Open to suggestions here.

Reply:
Another option is to set the default to `true`, then do the inverse check for Laravel version, thereby falling back to `false` if the config option is not set _and_ the Laravel version is less than 9.